package.json

```diff
@@ -65,9 +65,10 @@
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
     "prop-types": "^15.6.0",
-    "react": "^15.6.2",
+    "raf": "^3.4.0",
```

Comment:
just wondering - why is this required?

Reply:
It's a polyfill for `requestAnimationFrame` that React 16 relies on. `raf` is recommended since it supports the polyfill in jest testing that is also required.

Here's a link to the v16 release notes that talks about it https://reactjs.org/blog/2017/09/26/react-v16.0.html#javascript-environment-requirements